Source/SIMPLib/DataArrays/DataArray.hpp

```diff
@@ -1380,19 +1380,28 @@ template <typename T> class DataArray : public IDataArray
 
     QString info;
     QTextStream ss(&info);
-    if(format == SIMPL::HtmlFormat)
+
+    QString bgColor = "";
+    QString nameBgColor = "";
+
+    switch(format)
+    {
+    case SIMPL::ToolTipFormat:
+      bgColor = "bgcolor=\"#FFFCEA\"";
+      nameBgColor = "bgcolor=\"#E9E7D6\"";
+    case SIMPL::HtmlFormat:
```

Comment:
Add a comment here (before the `case SIMPL::HtmlFormat:`) stating that the fall-through is intentional.  Better yet, use a commented out `[[fallthrough]]` attribute to document the intention.  Without any kind of documentation of intent, this looks like a mistake and could be easily "fixed" while trying to reduce compiler warnings.